ops/charm.py

```diff
@@ -998,6 +1011,8 @@ class SecretExpiredEvent(SecretEvent):
     This event is fired on the secret owner to inform it that the secret revision
     must be removed. The event will keep firing until the owner removes the
     revision by calling :meth:`event.secret.remove_revision() <ops.Secret.remove_revision>`.
+
+    *Added in Juju 3.0*
```

Comment:
I like this formatting the most (though I'd want a full stop at the end of the sentence -- Python docs have this too).

The "tip" definitely isn't right. The "note" is okay but the formatting is very big/heavy. So the plain italics seems best.

As you note, the `versionadded` directive leads to funny formatting with "New in version Juju". Also, it seems the intention with "versionadded" is for the version of *this* project, that is, Ops itself, so if we were going to add those we'd use it for that (but we've decided not to).

So I'd go with the plain italics: `*Added in Juju 3.0.*`

Alternatively, can we write our own `jujuversion` directive? Looks like we might be able to do that *relatively* easily: https://www.sphinx-doc.org/en/master/development/tutorials/extending_syntax.html ... not sure whether it's worth it over just an italics phrase though.

Reply:
> Alternatively, can we write our own `jujuversion` directive? Looks like we might be able to do that _relatively_ easily: https://www.sphinx-doc.org/en/master/development/tutorials/extending_syntax.html ... not sure whether it's worth it over just an italics phrase though.

I somehow only just found this comment, which you had indeed made before mentioning it in the daily as well. Sorry for that!